packages/e2e-test-app-fabric/windows/RNTesterApp-Fabric/RNTesterApp-Fabric.cpp

```diff
@@ -0,0 +1,283 @@
+// RNTesterApp-Fabric.cpp : Defines the entry point for the application.
+//
+
+#include "pch.h"
+#include "RNTesterApp-Fabric.h"
+
+#include "../../../../vnext/codegen/NativeDeviceInfoSpec.g.h"
+
+#include <DispatcherQueue.h>
+#include <UIAutomation.h>
+
+#include <winrt/Microsoft.ReactNative.Composition.h>
+#include <winrt/Windows.UI.Composition.Desktop.h>
+
+#include "NativeModules.h"
+#include "ReactPropertyBag.h"
+
+#define MAX_LOADSTRING 100
+#define BTN_ADD 1000
+
+// Work around crash in DeviceInfo when running outside of XAML environment
+// TODO rework built-in DeviceInfo to allow it to be driven without use of HWNDs or XamlApps
+REACT_MODULE(DeviceInfo)
+struct DeviceInfo {
+  using ModuleSpec = Microsoft::ReactNativeSpecs::DeviceInfoSpec;
+
+  REACT_INIT(Initialize)
+  void Initialize(React::ReactContext const &reactContext) noexcept {
+    m_context = reactContext;
+  }
+
+  REACT_GET_CONSTANTS(GetConstants)
+  Microsoft::ReactNativeSpecs::DeviceInfoSpec_Constants GetConstants() noexcept {
+    Microsoft::ReactNativeSpecs::DeviceInfoSpec_Constants constants;
+    Microsoft::ReactNativeSpecs::DeviceInfoSpec_DisplayMetrics screenDisplayMetrics;
+    screenDisplayMetrics.fontScale = 1;
+    screenDisplayMetrics.height = 1024;
+    screenDisplayMetrics.width = 1024;
+    screenDisplayMetrics.scale = 1;
+    constants.Dimensions.screen = screenDisplayMetrics;
+    constants.Dimensions.window = screenDisplayMetrics;
+    return constants;
+  }
+
+ private:
+  winrt::Microsoft::ReactNative::ReactContext m_context;
+};
+
+// Have to use TurboModules to override built in modules.. so the standard attributed package provider doesn't work.
```

Comment:
This is no longer true. You can use the standard AddAttributedModules and pass true for make them turbomodules

Reply:
Should probably fixup the playground-comp too.

Reply:
https://github.com/microsoft/react-native-windows/issues/11413